### PR TITLE
[Identity] Garnett hijinks

### PR DESCRIPTION
**Pull request by walaura**
Adds garnett sprinkles all over identity on frontend. Tested `/email-prefs`, `/consents`, `/validate-email` and `/reauthenticate`.

- [x] Should there a more generic source for the header variables? new header is loaded separately and header.identity can't pick them up afaik.
- [x] Is `$identity-main` as a colour a go for the 15th?
- [x] Does it look pretty? ✨

### CODE REVIEW

File: static/src/stylesheets/module/identity/_header.garnett.scss [hunk 1]
```diff
@@ -0,0 +1,132 @@
+$gutter: 6px;
+$height: $identity-header-height - ($gutter * 2);
+
+$nav-background-colour: mix($sport-garnett-main-1, $neutral-7, 4%); // BG of the nav & menu
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/identity/_header.garnett.scss]: Why don't you just add an import at the top of this file for https://github.com/guardian/frontend/blob/master/static/src/stylesheets/layout/garnett-header/_variables.scss ?

Comment (walaura) [reply static/src/stylesheets/module/identity/_header.garnett.scss]: I did not know i could do that. thanks!!

**Review by mariogalic — APPROVED** [review]

**Comment by NataliaLKB** [comment]
Somehow a commit called `trivial` never makes me think its trivial 😂

**Comment by walaura** [comment]
@NataliaLKB haha, that reset got lost in the way, I'm double checking everything and this was the only messup

File: static/src/stylesheets/module/identity/_wrapper.garnett.scss [hunk 2]
```diff
@@ -0,0 +1,77 @@
+//TODO: most of this is not actually needed post-switch it's just overrides
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/identity/_wrapper.garnett.scss]: I would make this fairly explicit maybe:
`Post garnett launch (January 15th, 2018) most of this wont be needed/will need to be refactored as it is all overrides`

**Review by NataliaLKB — APPROVED** [review]
Looking forward to seeing this!!

**Pull request merged**